server/app/service/ses_client.js

```diff
@@ -39,9 +33,9 @@ const sendEmail = (to, from, subject, message) => {
 
     ses.sendEmail(params, (err, data) => {
         if (err) {       
-            return console.log(err, err.stack);
+            return logger.error("%o %o",err, err.stack);                
         } else {
-            console.log("Email sent.", data);        
+            logger.info("Email sent. %O ", data)
```

Comment:
log only required data. there is a chance that critical information will get logged. Log to and from address and subject etc. do not log body